boards/s32k144_evb/layout.ld

```diff
@@ -1,14 +1,16 @@
 /* Memory layout for the NXP S32K144
- * rom = 64KB (LENGTH = 0x00010000)
- * kernel = 128KB
- * user = 128KB
- * ram = 1024KB */
+ * rom =
+ * kernel = 
+ * user = 
+ * ram =  */
 
 MEMORY
 {
-  rom (rx)  : ORIGIN = 0x00000000, LENGTH = 0x00020000
-  prog (rx) : ORIGIN = 0x00020000, LENGTH = 0x00020000
-  ram (rwx) : ORIGIN = 0x20000000, LENGTH = 1024K
+  ivt (rx)  : ORIGIN = 0x00000000, LENGTH = 0x400
+  fcf (ro)  : ORIGIN = 0x00000400, LENGTH = 0x10
+  rom (rx)  : ORIGIN = 0x00000410, LENGTH = 0x00020000
+  prog (rx) : ORIGIN = 0x00020000, LENGTH = 0x00070000
```

Comment:
This will cause the flash area to end at 0x90000, but the S32K144 only has 0x80000 of flash.

Reply:
prog (rx) : ORIGIN = 0x00020000, LENGTH = 0x00060000